src/components/CvViewer/Templates/Basic/Basic.tsx

```diff
@@ -1,17 +1,18 @@
-import { ResumeViewerType } from '../../CvViewer.types';
 import css from './Basic.module.css';
 import { TextWithHeading } from '../../TextWithHeading/TextWithHeading';
 import { CVExperienceBlock } from '../../WorkBlock/CVExperienceBlock';
 import { StringArrayViewer } from '../../StringArrayViewer/StringArrayViewer';
 import { Avatar } from '../../Avatar/Avatar';
 import { PersonalInfo } from '../../PersonalInfo/PersonalInfo';
+import { useAppSelector } from '../../../../hooks/redux';
+import { getDateFromTimeStampInDay } from '../../formatters/date.formatters';
 
 type CvViewerProps = {
-  cv: ResumeViewerType;
   isReverse?: boolean;
 };
 
-export const Basic = ({ cv, isReverse = false }: CvViewerProps) => {
+export const Basic = ({ isReverse = false }: CvViewerProps) => {
+  const { cv, chat } = useAppSelector((state) => state);
```

Comment:
Let's not reach for the state inside CvComponent, every other change you would need to rewrite something in every template, CvComponent should take solid defined state from one entry point, and then it's only responsibility should be the rendering